build/references/coreth-arc20s.md

```diff
@@ -0,0 +1,103 @@
+# Coreth ANTs and ARC20s
+
+## ANTs on the C-Chain
+
+### Why move an ANT from the X-Chain to the C-Chain?
+
+An Avalanche Native Token (ANT) is any token created on the X-Chain. These tokens can be exchanged on the X-Chain at lightning fast speeds taking advantage of the superior performance offered by using a DAG instead of a strictly linear chain.
+
+When we want to use smart contract functionality, we need a total ordering of state transitions (transactions), so that each transaction can modify the state of the network without having to worry about external dependencies.
+
+Therefore, when we want to use smart contract functionality, we must perform an atomic swap to move ANTs from the X-Chain to the C-Chain.
+
+### ANTs on the C-Chain
+
+#### Native Coin - AVAX
+
+ANTs on the C-Chain are either AVAX &emdash; the Avalanche native asset &emdash; or another asset that was created on the X-Chain.
+
+The C-Chain is a modified instance of the EVM, so AVAX on the C-Chain is equivalent to ETH on the Ethereum network. When you create or call a smart contract, you pay for gas costs with AVAX. You can natively transfer AVAX between accounts, send it to a smart contract, all using native EVM tools and libraries.
+
+#### EVM Transaction
+
+An EVM Transaction is composed of the following fields:
+
+* **`nonce`** Scalar value equal to the number of transactions send by the sender
+* **`gasPrice`** Scalar value equal to tthe number of Wei paid per unit of gas to execute this transaction
```

Comment:
tthe

Reply:
Do we define how many wei == 1 nAVAX?

Reply:
1 wei = 1 nAVAX * 10^-9. I think it will be better to use the wei convention here because atm that is how all of the tools will display the information. I'll add in a conversion though.